idb/postgres/postgres.go

```diff
@@ -174,9 +178,15 @@ func (db *IndexerDb) init(opts idb.IndexerDbOptions) (chan struct{}, error) {
 }
 
 // AddBlock is part of idb.IndexerDb.
-func (db *IndexerDb) AddBlock(vb *ledgercore.ValidatedBlock) error {
-	block := vb.Block()
-	db.log.Printf("adding block %d", block.Round())
+func (db *IndexerDb) AddBlock(vblk *ledgercore.ValidatedBlock) error {
+	//todo: use a converter util until vblk type is changed
```

Comment:
Not for this PR, but maybe an incremental next step could be changing the interface and doing the conversion one layer up.